src/lib/Telemetry/telemetry.h

```diff
@@ -74,5 +76,6 @@ class Telemetry
     bool callEnterBind;
     bool callUpdateModelMatch;
     bool sendDeviceFrame;
+    bool crsfBatterySensorDetected;
```

Comment:
I could be wrong, but its probably not the best practice to assume this bool will initialise to false...

Reply:
C++ guarantees that variables are created with a specific initial value. And for booleans that is the `false` value 😄

Reply:
Thats not what 5mins on stackoverflow taught me, but fair enough. I was always told that member vars should always be initialised (i.e. in the class constructor etc.)

Reply:
https://stackoverflow.com/questions/4622225/arent-boolean-variables-always-false-by-default

Reply:
Im happy to change it just for the readability.  But there are multiple instances that should be updated and probably not as part of this PR.

Reply:
Yep, you are right Wez, only globals are automatically set to defaults.